docs/admin/admission-controllers.md

```diff
@@ -414,9 +414,9 @@ This admission controller also protects the access to `metadata.ownerReferences[
 of an object, so that only users with "update" permission to the `finalizers`
 subresource of the referenced *owner* can change it.
 
-### Storage Protection (beta)
+### Storage Object in Use Protection (beta)
 {% assign for_k8s_version="v1.10" %}{% include feature-state-beta.md %}
-The `StorageProtection` plugin adds the `kubernetes.io/pvc-protection` or `kubernetes.io/pv-protection` finalizers to newly created Persistent Volume Claims (PVCs) or Persistent Volumes (PV). In case a user deletes a PVC or PV the PVC or PV is not removed until the finalizer is removed from the PVC or PV by PVC or PV Protection Controller. Refer to the [PVC Protection](/docs/concepts/storage/persistent-volumes/#persistent-volume-claim-protection) for more detailed information.
+The `StorageObjectInUseProtection` plugin adds the `kubernetes.io/pvc-protection` or `kubernetes.io/pv-protection` finalizers to newly created Persistent Volume Claims (PVCs) or Persistent Volumes (PV). In case a user deletes a PVC or PV the PVC or PV is not removed until the finalizer is removed from the PVC or PV by PVC or PV Protection Controller. Refer to the [Storage Object in Use Protection](/docs/concepts/storage/persistent-volumes/#storage-object-in-use-protection) for more detailed information.
```

Comment:
`/docs/concepts/storage/persistent-volumes/#storage-object-in-use-protection` is not a right path

Reply:
we have PVC protection doc as well as PV protection doc, should we merge them into one ?

Reply:
Well it behaves strangely for me. Any links to any 3rd level headers didn't work for me, e.g. [this one](https://deploy-preview-7576--kubernetes-io-vnext-staging.netlify.com/docs/concepts/storage/persistent-volumes/#provisioning) or [that one](https://deploy-preview-7576--kubernetes-io-vnext-staging.netlify.com/docs/admin/admission-controllers/#alwayspullimages).
Afterwards, I tried to open it in incognito window and it worked. So I cleared all my browsing files and images and it works also. So you may also have some conflicts with cached files.